admin_manual/configuration_server/config_sample_php_parameters.rst

```diff
@@ -1270,6 +1270,8 @@ following conditions are met Nextcloud URLs won't contain `index.php`:
 
 Defaults to ``''`` (empty string)
 
+.. _config-php-sample-htaccess-ignorefrontcontroller:
```

Comment:
I'm afraid you cannot add a label here because the section is generated, and it will be overwritten.